### PR TITLE
missing zero fields coppa,instl,allimp,test

### PR DESCRIPTION
**Pull request by trilopin**
We want to send explicitly 0 for test, allimp and instl even though they are 0 by default and could be ommitted.

Coppa, on the other hand, is not default 0 and should be treated as a pointer (like gdpr)

### CODE REVIEW

**Review by MihaiLupoiu — APPROVED** [review]

**Review by ajimenezherrero — APPROVED** [review]
LGTM

File: openrtb2/bid_request.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ type BidRequest struct {
 	// Description:
 	//    Indicator of test mode in which auctions are not billable,
 	//    where 0 = live mode, 1 = test mode.
-	Test int8 `json:"test,omitempty"`
```
Comment (blueonrails) [inline openrtb2/bid_request.go:93]: should we add some comment saying this behavior is Seedtag's custom?

**Review by blueonrails — APPROVED** [review]

File: openrtb2/bid_request.go (hunk 1 above)
Comment (trilopin) [reply openrtb2/bid_request.go:93]: added a document explaining changes done and workflow

Comment (trilopin) [reply openrtb2/bid_request.go:93]: added a document

**Pull request merged**